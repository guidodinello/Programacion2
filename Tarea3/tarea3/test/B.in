# Prueba B
insertarEnBinario (16,16.16)
insertarEnBinario (8,8.8)
insertarEnBinario (24,24.24)
insertarEnBinario (4,4.4)
insertarEnBinario (12,12.12)
nivelEnBinario 0
nivelEnBinario 1
nivelEnBinario 2
menores 0
menores 16
esAvl
removerDeBinario 4
esCamino 3 (16,1) (24,1) (28,1)
esCamino 2 (16,1) (24,1)
Fin