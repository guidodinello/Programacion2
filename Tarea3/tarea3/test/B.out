1>#  Prueba B.
2>Insertado.
3>Insertado.
4>Insertado.
5>Insertado.
6>Insertado.
7>l = 0.
8>(16,16.16)
9>(8,8.80)(24,24.24)
10>
11>
-(12,12.12)
(8,8.80)
-(4,4.40)
12>AVL.
13>Removido.
14>NO es camino.
15>Es camino.
16>Fin.
